src/hermit.rs

```diff
@@ -43,6 +43,24 @@ impl<T: Config> Hermit<T> {
         let path = self.config.shell_root_path().join(name);
         file_ops.create_git_repo(path);
     }
+
+    pub fn inhabit_shell(&mut self, file_ops: &mut FileOperations, name: &str) -> Result<(), Error> {
+        //let mut shell_names = Vec::new();
+        //let mut res = Ok(());
+        //if name == "" {
+        //    shell_names = self.get_shell_list();
+        //} else {
```

Comment:
I'm sure you're all over this, but definitely remove this commented out code before this gets merged.